entrypoint.sh

```diff
@@ -13,7 +13,7 @@ set +x
 #test $DATABASE_USER
 #test $DATABASE_PASSWORD
 #test $IPFS_INIT
-VDB_COMMAND=${VDB_COMMAND:-watch}
+#VDB_COMMAND=${VDB_COMMAND:-watch}
 set +e
```

Comment:
I think we should get rid of this `set +e` and the `set -e` on line 5, and get rid of the commented out stuff in-between them.